### PR TITLE
Update terms

### PR DESCRIPTION
**Pull request by LauraLangdon**
Last week we changed:

- access key -> API key
- editor token -> session token

But these changes were only made in the quickstart, so this PR makes those changes globally (on the beta docs branch).

Also need new screenshots of the screens of the admin dashboard, once the terms are updated there as well, which I can do once https://github.com/suborbital/se2/pull/1169 is merged.

### CODE REVIEW

**Comment by LauraLangdon** [comment]
Bump! @callahad 😊

File: website/docs/reference/future-glossary.md [hunk 1]
```diff
@@ -14,4 +14,4 @@ pagination_next: null
 | Tenant          | words | Identifies the relationship between an Environment and some specific user or collection of users interacting with it. The Environment administrator manages and identifies this relationship including, but not limited to, tenant authentication.                                                                                                                                             |   tnt                         |
 | Client | words | The application communicating with SE2 on the behalf of some Tenant |                            |
 | Environment Token     | words | Grants a Client access to an environment    |                            |
-| Editor Token  | words | Grants a Tenant access to the Builder Service                                                                                                                                     |                            |
+| Session  Token  | words | Grants a Tenant access to the Builder Service                                                                                                                                     |                            |
```
Comment (javorszky) [inline website/docs/reference/future-glossary.md]: tiny tiny nit, not blocker, but there are two spaces here

**Review by javorszky — APPROVED** [review]
One very very tiny result

File: website/docs/reference/future-glossary.md (hunk 1 above)
Comment (LauraLangdon) [reply website/docs/reference/future-glossary.md]: Ah, cheers!

**Pull request merged**